core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4204,6 +4204,15 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey WORKER_NIO_DIRECT_BUFFER_POOL_ENABLED =
+      booleanBuilder(Name.WORKER_NIO_DIRECT_BUFFER_POOL_ENABLED)
+          .setDefaultValue(true)
```

Comment:
As this is a new coming feature, would you like to set the default value to false?